### PR TITLE
Javascript classes

### PR DESCRIPTION
**Pull request by adelphinsucasa**
In this PR I fullfilled: 
[ x ] Implement BookCollection Class
[ x ] Add AddBook method
[ x ] Add RemoveBook method
[ x ] Add Setter and Getter methods
[ x ] Add Styles CSS file

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    const newBook = {
+      title: cTitle,
+      author: cAuthor,
+    };
+    this.books.push(newBook);
+    localStorage.setItem('localStorageBooks', JSON.stringify(this.books));
+  }
+
+  removeBook(index) {
+    const result = this.books.filter((book) => book.title !== this.books[index].title);
```
Comment (brytebee) [inline script.js]: - [x] **Required:** Currently when I enter a book multiple times with the same title and one is removed, all of the books with that title are removed. Kindly fix this.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @adelphinsucasa & @chelobotix👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (adelphinsucasa) [reply script.js]: Hi brytebee.
You are right, we were filtering by the books title. Now we are filtering by the index of the array. 
Thank you very much for your review

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved  🟢 🟢 🟢

Hi Team,

Good job working on the changes asked by the previous reviewer 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:
✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used JS classes and methods 👍🏽
✅ Add and remove features work fine as expected 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**